docs/organizations/using-coding-standards.md

```diff
@@ -108,6 +103,25 @@ To delete a coding standard:
 
     ![Deleting a coding standard](images/coding-standard-delete.png)
 
+## Defining a default coding standard {: id="set-default"}
+
+For ease of management, you can define a default coding standard that automatically applies to new repositories. In the absence of a default coding standard, Codacy applies global defaults to all new repositories.
```

Comment:
(caveat that might be worth meantioning: when Codacy introduces new tools those are automatically added to the global default but need to be manually added to the default coding standard)